tools/update_file.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
```

Comment:
This doesn't look like a directly invokable file? Above, it's used as a library. I don't see the code that directly invokes it (i.e. where name is `__main__`).

As such, please remove the shebang, if-main and main() from this file.

Reply:
Good catch, it was a remnant of the times when the `update_pip_deps.py` was a shell script. Removed.